CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ option(BUILD_EXAMPLES "Build examples" ${BUILD_PROGRAMS})
 option(BUILD_ORM "Build orm" ON)
 option(COZ_PROFILING "Use coz for profiling" OFF)
 option(BUILD_DROGON_SHARED "Build drogon as a shared lib" OFF)
+option(BUILD_DOC "Build Doxygen documentation" FALSE)
```

Comment:
Use `OFF` to keep it consistent

Reply:
Sorry, it's corrected.

Reply:
No pressure. I also make these mistakes all the time :smiley: